servers/display_server.h

```diff
@@ -308,6 +308,22 @@ class DisplayServer : public Object {
 		}
 		return scale;
 	}
+
+	enum IOSUIGestureState {
+		IOS_DEFAULT,
+		IOS_SUPPRESS_GESTURE_ALL,
+		IOS_INDICATOR_HIDE,
+		IOS_SUPPRESS_GESTURE_HOME_ONLY,
+		IOS_SUPPRESS_GESTURE_STATUS_ONLY,
+		IOS_SUPPRESS_GESTURE_STATUS_ONLY_INDICATOR_HIDE,
+	};
+
+	virtual IOSUIGestureState ios_get_gesture_state() const;
+	virtual void ios_set_gesture_state(IOSUIGestureState ios_gesture_set);
```

Comment:
```suggestion
	virtual void ios_set_gesture_state(IOSUIGestureState p_ios_gesture_set);
```
Prefix arguments with `p_`